### PR TITLE
[5.0] Upgrade to MPL 2.0

### PR DESCRIPTION
**Pull request by jamesaimonetti**
- add MPL 2.0 to all core/app folders for breakening
- update all source files to include MPL license info

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -280,6 +280,7 @@ app_applications:
 	ERL_LIBS=deps:core:applications $(ROOT)/scripts/apps_of_app.escript -a $(shell find applications -name *.app.src)
 
 code_checks:
+	@$(ROOT)/scripts/bump-copyright-year.sh $(CHANGED_ERL)
```
Comment (icehess) [inline Makefile]: edocify script is bumping the copyright year too. So can we add adding license line to edocify instead?

Comment (jamesaimonetti) [reply Makefile]: what's the advantage?

Comment (icehess) [reply Makefile]: None.

Just as you know with this change now both edocify and bump-copyright-year.sh are trying to bump the year.

**Comment by jamesaimonetti** [comment]
@icehess i think all are caught now. Had to add exception for amqp_cron but otherwise...
Also don't think the analyze portion will be fixable :p No code changes though so I think it can be ignored.

**Pull request merged**